src/TinyGsmClientSIM70xx.h

```diff
@@ -90,6 +90,10 @@ class TinyGsmSim70xx : public TinyGsmModem<TinyGsmSim70xx<modemType>>,
     if (thisModem().waitResponse(5000L, res2) != 1) { return name; }
     res2.replace(GSM_NL "OK" GSM_NL, "");
     res2.replace("_", " ");
+    if (res2.startsWith("AT+GMM")) {
+      // https://github.com/Xinyuan-LilyGO/LilyGo-T-SIM7080G/issues/44
+      res2 = res2.substring(6);
+    }
```

Comment:
fix for https://github.com/Xinyuan-LilyGO/LilyGo-T-SIM7080G/issues/44

Reply:
AT+GMM is related to whether AT echo is turned on. If the echo is turned on, the data obtained will carry the sending command. The way to solve this problem is to turn off the echo first. This is not actually a problem, because the regular TinyGSM starts by default. In all cases, AT echo is turned off first by calling init.